workflows/funcscan.nf

```diff
@@ -156,16 +160,20 @@ workflow FUNCSCAN {
 
         if ( params.annotation_tool == "prodigal" ) {
             PRODIGAL_GFF ( ch_prepped_input, "gff" )
+            GUNZIP_FAA ( PRODIGAL_GFF.out.amino_acid_fasta )
+            GUNZIP_FNA ( PRODIGAL_GFF.out.nucleotide_fasta)
+            GUNZIP_GFF ( PRODIGAL_GFF.out.gene_annotations )
             ch_versions              = ch_versions.mix(PRODIGAL_GFF.out.versions)
-            ch_annotation_faa        = PRODIGAL_GFF.out.amino_acid_fasta
-            ch_annotation_fna        = PRODIGAL_GFF.out.nucleotide_fasta
-            ch_annotation_gff        = PRODIGAL_GFF.out.gene_annotations
+            ch_annotation_faa        = GUNZIP_FAA.out.gunzip
+            ch_annotation_fna        = GUNZIP_FNA.out.gunzip
+            ch_annotation_gff        = GUNZIP_GFF.out.gunzip
             ch_annotation_gbk        = Channel.empty() // Prodigal doesn't produce GBK
```

Comment:
Prodigal doesn't produce GBK stimmt ja eigentlich nicht ganz ...
```suggestion
            ch_annotation_gbk        = Channel.empty() // Prodigal produces either GFF or GBK
```

Reply:
I addressed that in a previous PR, but thanks for noticing. Will be resolved after merge conflict resolving :+1: